hivemq-edge/src/main/java/com/hivemq/api/auth/provider/IUsernameRolesProvider.java

```diff
@@ -15,15 +15,23 @@
  */
 package com.hivemq.api.auth.provider;
 
+import com.hivemq.api.auth.ApiPrincipal;
 import org.jetbrains.annotations.NotNull;
-import com.hivemq.http.core.UsernamePasswordRoles;
 
 import java.util.Optional;
+import java.util.Set;
 
 /**
  * @author Simon L Johnson
```

Comment:
please remove all references to Simon L Johnson from the files. There are a number of files that were produced by pasting a formerly existing file, with a new name, then the content was removed and edited (I suspect), but the author comment was left behind.